src/config.hpp

```diff
@@ -52,7 +57,7 @@ class counter_config {
 class config {
 public:
     // Setup default configurations.
-    config(): m_vip("127.0.0.1"), m_tempdir(DEFAULT_TMPDIR) {
+    config(): m_vip(std::optional("127.0.0.1")), m_tempdir(DEFAULT_TMPDIR) {
```

Comment:
`m_leader_election_method` is initialized with the following line:

```cpp
yrmcds::leader_election_method m_leader_election_method = yrmcds::leader_election_method::virtual_ip;
```

Reply:
I know. I thought it was inconsistent.

Reply:
I will move all member initializations to the declaration statement.
I think it is easier to read when they are grouped together rather than separated in the constructor and member declarations.